data/block_export_data.go

```diff
@@ -14,8 +15,12 @@ type RoundProvider interface {
 
 // BlockData is provided to the Exporter on each round.
 type BlockData struct {
-	// Block is the block data written to the blockchain.
-	Block *bookkeeping.Block
+
+	// BlockHeader is the immutable header from the block
+	BlockHeader bookkeeping.BlockHeader
+
+	// Payset is the set of data the block is carrying--can be modified as it is processed
+	Payset *transactions.Payset
```

Comment:
Does this need to be a pointer?  The Payset is a slice so it can have a len of 0 by default.